app/components/post_draft/quick_actions/quick_actions.tsx

```diff
@@ -96,6 +108,13 @@ export default function QuickActions({
                 testID={cameraActionTestID}
                 {...uploadProps}
             />
+            {isPostPriorityEnabled && location !== Screens.THREAD && (
```

Comment:
I've tracked the location prop all the way up to here, and I think the best way to deal with this is to change the property to be a boolean, something called `canSetPriority` or something like that